README.md

```diff
@@ -115,42 +113,93 @@ mvn install #inside the spring-search folder
     <artifactId>spring-search</artifactId>
     <version>0.0.4-SNAPSHOT</version>
 </dependency>
+```
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
 
+Create a Rest repository extending JpaSpecificationExecutor
```

Comment:
Your repository should be annotated as a `RepositoryRestResource` and should extend `JpaSpecificationExecutor`